pipeline/src/simulate/simulate_prefix.jl

```diff
@@ -0,0 +1,12 @@
+"""
+Internal method for setting the prefix for the truth data file name.
+"""
+_simulate_prefix(pipeline::AbstractEpiAwarePipeline) = "truth_data"
```

Comment:
It seems like you could do this via being a bit clever with the type name in the default method and then not need all this boiler plate?